src/views/Search.vue

```diff
@@ -86,11 +86,13 @@ import TextInput from '@/components/TextInput.vue'
 import DropdownSelect from '@/components/DropdownSelect.vue';
 import { useRouter } from 'vue-router';
 import { useAppStore } from '@/store/app'
+import JSONbigint from 'json-bigint'
```

Comment:
I use something similar in https://github.com/sdss/zora/blob/a925ff46da5b434f160df2376d2aa6e20eac253c/src/views/Target.vue#L103

Reply:
I'm happy to keep `json-big`. They seem to do the same thing.